user-console/src/main/java/org/pentaho/mantle/client/admin/ChangePasswordByUserDialog.java

```diff
@@ -155,10 +159,30 @@ public void onClick( ClickEvent event ) {
         disableAcceptBtn();
         String newPassword = newPasswordTextBox.getText();
         String oldPassword = oldPasswordTextBox.getText();
+        String allowedChars = "a-z A-Z 0-9 _ . , : ; < > | ! @ # $ % ^ & * ( ) [ ]";
```

Comment:
This should be a static final and should be declared near the "regex variables" in line 50/51.